can/interfaces/vector/canlib.py

```diff
@@ -484,6 +486,7 @@ def _detect_available_configs():
                     "interface": "vector",
                     "app_name": None,
                     "channel": channel_config.channelIndex,
+                    'supportsFd': bool(channel_config.channelBusCapabilities & vxlapi.XL_CHANNEL_FLAG_CANFD_ISO_SUPPORT)}
```

Comment:
This should probably be like that, to be consistent with the naming scheme:

```suggestion
                    'supports_fd': bool(channel_config.channelBusCapabilities & vxlapi.XL_CHANNEL_FLAG_CANFD_ISO_SUPPORT)}
```